File: database/src/main/java/com/pelisat/cesp/ceemsp/database/dto/metadata/VisitaArchivoMetadata.java
package com.pelisat.cesp.ceemsp.database.dto.metadata;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class VisitaArchivoMetadata {
    private int id;
    private String uuid;
    private String nombreArchivo;
    private String descripcion;
}

File: ceemsp-frontend/src/app/_models/VisitaArchivoMetadata.ts
import {Deserializable} from "./Deserializable";

export default class VisitaArchivoMetadata implements Deserializable {
  id: number;
  uuid: string;
  nombreArchivo: string;
  descripcion: string;

  deserialize(input: any): this {
    Object.assign(this, input);
    return this;
  }
}
